frontend/src/components/MapView/DateSelector/TimelineItems/TimelineLabel/__snapshots__/index.test.tsx.snap

```diff
@@ -6,7 +6,7 @@ exports[`TimelineLabel renders as expected 1`] = `
     classname="TimelineLabel-dateItemLabel-1"
     variant="body2"
   >
-    juin 2023
+    juillet 2023
```

Comment:
this means that one of our tests doesn't have a properly mocked date @HarryMytilinaios

Reply:
Yes I think so this test was failing for the TimelineLabels before I push my code. I think when this test was created we had june and the props we are passing set a new Date() with the appropriate locale this is the reason for this. Do you want me to change it?

Reply:
yes, let's find where this prop is getting set and fix the date in jest if necessary

Reply:
You can see in `/DateSelector/index.test.tsx` how we "fix" the date